modules/investigating-kernel-crashes.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operating-system-issues.adoc
+
+[id="investigating-kernel-crashes"]
+= Investigating kernel crashes
+
+== Enabling kdump
+
+`kdump` (included in `kexec-tools`) is a service providing a crash dumping mechanism. The service enables you to save the contents of the system’s memory for later analysis.
+
+{op-system} ships with `kexec-tools`, but manual configuration is required to enable `kdump`.
+
+The following are the basic steps needed to enable `kdump` on {op-system}.
+
+.  Memory must be reserved for the crash kernel during booting of the first kernel. Kernel arguments can be provided like this:
++
+[source, terminal]
+----
+sudo rpm-ostree kargs --append='crashkernel=256M'
+----
+
+.  By default, the path in which the vmcore will be saved is `/var/crash`. It is also possible to write the dump over the network or to some other location on the local system by editing `/etc/kdump.conf`. For additional information, see `kdump.conf(5)` and the comments in `/etc/kdump.conf` and `/etc/sysconfig/kdump`.
```

Comment:
Would you mind providing an example for editing `/etc/kdump.conf`?

For the additional information portion, following the example this can be nested in the step. 
. Step
+
`Example`
+
Additional info
+
. Next step

And for the content of the additional info, I'm not sure what the (5)/(8) indicate here, but maybe:

For additional information, see `kdump.conf`, a manual page for the /etc/kdump.conf configuration file containing the full documentation of available options, and the comments in `/etc/kdump.conf` and `/etc/sysconfig/kdump`. ifdef::openshift-enterprise[]
Also refer to the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/installing-and-configuring-kdump_system-design-guide#configuring-the-kdump-target_configuring-kdump-on-the-command-line[RHEL `kdump` documentation] for further information on configuring the dump target.
endif::[]